include/zephyr/xen/dom0/zimage.h

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
(note) drop this file here and move it to zephyr zenlib 
@firscity @oleksiimoisieiev FYI